### PR TITLE
Current metrics: Add swap usage

### LINKED ISSUE #26: add current swap usage
See design: https://garrett.github.io/cockpit-mockweb/metrics/current/

Cockpit metrics have `memory.swap-used`. Nothing for swap size, we need to get that from /proc/meminfo `SwapTotal`. Incidentally, `cpu_ram_info()` already does that.

### PR DESCRIPTION
**Pull request by martinpitt**
Only show the swap bar if the machine actually has any swap.

Fixes #26

### CODE REVIEW

**Comment by martinpitt** [comment]
The layout of that is still a bit weird, but it will change in PR #21 and PR #27 anyway:

image

Also, the label and current value are misaligned, this looks like the same problem as in PR #24 -- @garrett wanted to take a look at this.

Thus I propose to ignore these for now, and just land the functionality. The whole thing will need another design scrubbing anyway.

**Comment by martinpitt** [comment]
Fedora 33 seems to [handle swap differently](https://logs.cockpit-project.org/logs/pull-29-20200907-093514-bf0ffa29-fedora-33/log.html#2-2) -- AFAIR it moved to zswap, so `swapoff --all` does not seem to work any more. Investigating..

**Comment by martinpitt** [comment]
zram doesn't behave fundamentally different, it's just that `swapoff --all` is not reliable -- if enough swap is being used, it can't be cleanly removed apparently. I changed the test case to hopefully be more robust now.

File: test/check-application [hunk 1]
```diff
@@ -113,6 +113,8 @@ class TestApplication(testlib.MachineCase):
         b = self.browser
         m = self.machine
 
+        m.execute('swapoff --all; while [ -n "$(swapon --show)" ]; do sleep 1; done')
```
Comment (marusak) [inline test/check-application:116]: You had cleanup here before, won't it break non-d tests?

Comment (martinpitt) [reply test/check-application:116]: testDisks() is destructive -- that's the point, it needs to be a fresh boot without existing swap usage.

**Review by marusak — APPROVED** [review]
Oh right, I forgot that these tests are not non-d by default.
Thanks!

**Pull request merged**